#################
AnalysisTechnique
#################

Related schema specification: `AnalysisTechnique <https://openminds-documentation.readthedocs.io/en/latest/specifications/controlledTerms/analysisTechnique.html>`

------------

------------

3DComputerGraphicModeling
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/3DComputerGraphicModeling
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: 3D computer graphic modeling

`BACK TO TOP <AnalysisTechnique_>`_

------------

4PointsCongruentSetsAlignment
-----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/4PointsCongruentSetsAlignment
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 4-points congruent sets alignment is a fast and robust alignment technique for 3D point sets without pre-filtering or denoising the data, even if the data are noisy and/or contaminated with outliers ([Aiger et al., 2008](https://doi.org/10.1145/1360612.1360684)).
   :name: 4-points congruent sets alignment

`BACK TO TOP <AnalysisTechnique_>`_

------------

DNAMethylationAnalysis
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/DNAMethylationAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'DNA methylation analysis' studies chromosomal patterns of DNA or histone modification by methyl groups ([modified from Nature.com](https://www.nature.com/subjects/methylation-analysis)).
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0779582
   :name: DNA methylation analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

DNASequencing
-------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/DNASequencing
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'DNA sequencing' refers to a group of techniques that are used to determine the order of nucleotides (nucleic acid sequence) in DNA. [adapted from [wikipedia](https://en.wikipedia.org/wiki/DNA_sequencing)]
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0783031
   :name: DNA sequencing
   :preferredOntologyIdentifier: http://purl.obolibrary.org/obo/OBI_0000626

`BACK TO TOP <AnalysisTechnique_>`_

------------

GrubbsTest
----------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/GrubbsTest
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The 'Grubbs test' is a statistical test, first published by [Grubbs (1950)](https://doi.org/10.1214/aoms/1177729885), used to detect outliers in univariate data that are assumed to come from a normally distributed population. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Grubbs%27s_test)]
   :name: Grubbs' test

`BACK TO TOP <AnalysisTechnique_>`_

------------

HPCSimulation
-------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/HPCSimulation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: HPC simulation

`BACK TO TOP <AnalysisTechnique_>`_

------------

ICABasedDenoisingTechnique
--------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/ICABasedDenoisingTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: An 'ICA based denoising technique' removes independent components from input data to reduce noise while preserving the features of interest in the data.
   :name: ICA based denoising technique

`BACK TO TOP <AnalysisTechnique_>`_

------------

MannWhitneyUTest
----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/MannWhitneyUTest
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The 'Mann–Whitney U test' is a nonparametric test of the null hypothesis that, for randomly selected values X and Y from two populations, the probability of X being greater than Y is equal to the probability of Y being greater than X. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test)]
   :name: Mann–Whitney U test

`BACK TO TOP <AnalysisTechnique_>`_

------------

RNASequencing
-------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/RNASequencing
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'RNA sequencing' refers to a group of techniques that are used to (directly or indirectly) determine the order of nucleotides (nucleic acid sequence) in RNA.
   :interlexIdentifier: http://uri.interlex.org/ilx_0782092
   :name: RNA sequencing
   :preferredOntologyIdentifier: http://purl.obolibrary.org/obo/OBI_0001177

`BACK TO TOP <AnalysisTechnique_>`_

------------

ShapiroWilkTest
---------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/ShapiroWilkTest
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The 'Shapiro–Wilk test' is a statistical test of normality of a complete sample, first described by [Shapiro and Wilk (1965)](https://doi.org/10.1093/biomet/52.3-4.591). [adapted from [wikipedia](https://en.wikipedia.org/wiki/Shapiro%E2%80%93Wilk_test)]
   :name: Shapiro-Wilk test

`BACK TO TOP <AnalysisTechnique_>`_

------------

SpearmansRankOrderCorrelation
-----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/SpearmansRankOrderCorrelation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The 'Spearman's rank-order correlation' is the nonparametric version of the Pearson product-moment correlation measuring the strength and direction of association between a set of two ranked variables. [adapted from [Laerd.com](https://statistics.laerd.com/statistical-guides/spearmans-rank-order-correlation-statistical-guide.php)]
   :name: Spearman's rank-order correlation

`BACK TO TOP <AnalysisTechnique_>`_

------------

WardClustering
--------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/WardClustering
   :@type: https://openminds.ebrains.eu/controlledTerms/AnalysisTechnique
   :definition: 'Ward clustering' is a general agglomerative hierarchical clustering procedure, where the criterion for choosing the pair of clusters to merge at each step is based on the optimal value of an objective function (typically aiming to minimize the total within-cluster variance). [adapted from [Wikipedia](https://en.wikipedia.org/wiki/Ward%27s_method)]
   :name: Ward clustering

`BACK TO TOP <AnalysisTechnique_>`_

------------

activationLikelihoodEstimation
------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/activationLikelihoodEstimation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: An 'activation likelihood estimation' is a coordinate-based meta-analysis of neuroimaging data that determines the above-chance convergence of activation probabilities between experiments (i.e., not between foci). [adapted from [Eickhoff et al., 2011](https://dx.doi.org/10.1016%2Fj.neuroimage.2011.09.017)]
   :name: activation likelihood estimation

`BACK TO TOP <AnalysisTechnique_>`_

------------

affineImageRegistration
-----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/affineImageRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'affine image registration' is a process of bringing a set of images into the same coordinate system using affine transformation.
   :name: affine image registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

affineTransformation
--------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/affineTransformation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: An 'affine transformation' is a specific linear transformation using combinations of rotations, translations, reflections, scaling and shearing to map coordinates between two coordinate spaces.
   :name: affine transformation

`BACK TO TOP <AnalysisTechnique_>`_

------------

anatomicalDelineationTechnique
------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/anatomicalDelineationTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: anatomical delineation technique

`BACK TO TOP <AnalysisTechnique_>`_

------------

averageLinkageClustering
------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/averageLinkageClustering
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: average linkage clustering

`BACK TO TOP <AnalysisTechnique_>`_

------------

biasFieldCorrection
-------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/biasFieldCorrection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'bias field correction' is a mathematical technique to remove a corrupting, low frequency signal from magnetic resonance images. This bias field signal is typically caused by inhomogeneities in the magnetic ﬁelds of the magnetic resonance imaging machine.
   :name: bias field correction

`BACK TO TOP <AnalysisTechnique_>`_

------------

bootstrapping
-------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/bootstrapping
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: bootstrapping

`BACK TO TOP <AnalysisTechnique_>`_

------------

boundaryBasedRegistration
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/boundaryBasedRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The term 'boundary-based registration' refers to feature based image registration methods which utilize a boundary which can be identified in the source and target image.
   :name: boundary-based registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

clusterAnalysis
---------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/clusterAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: cluster analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

combinedVolumeSurfaceRegistration
---------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/combinedVolumeSurfaceRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The term 'combined volume-surface registration' refers to an image registration framework which utilizes information from the brain surface and the brain volume to perform the registration (cf. [Postelnicu et al. (2009)](https://doi.org/10.1109/TMI.2008.2004426)).
   :name: combined volume–surface registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

communicationProfiling
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/communicationProfiling
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: communication profiling

`BACK TO TOP <AnalysisTechnique_>`_

------------

conjunctionAnalysis
-------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/conjunctionAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: conjunction analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

connected-componentAnalysis
---------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/connected-componentAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'connected-component analysis' is an algorithmic application of graph theory, where subsets of connected components are uniquely labeled based on a given heuristic. [adapted from: [wikipedia](https://en.wikipedia.org/wiki/Connected-component_labeling)]
   :name: connected-component analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

connectivityBasedParcellationTechnique
--------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/connectivityBasedParcellationTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: connectivity based parcellation technique

`BACK TO TOP <AnalysisTechnique_>`_

------------

convolution
-----------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/convolution
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: In functional analysis, 'convolution' is a mathematical operation on two functions (f and g) producing a third function (f * g) that expresses how the shape of one is modified by the other. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Convolution)]
   :name: convolution

`BACK TO TOP <AnalysisTechnique_>`_

------------

correlationAnalysis
-------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/correlationAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: correlation analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

covarianceAnalysis
------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/covarianceAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: covariance analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

currentSourceDensityAnalysis
----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/currentSourceDensityAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: current source density analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

cytoarchitectonicMapping
------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/cytoarchitectonicMapping
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Cytoarchitectonic mapping' is a delineation technique that defines regional borders based on histological analysis of the cellular composition of the studied tissue.
   :name: cytoarchitectonic mapping

`BACK TO TOP <AnalysisTechnique_>`_

------------

deepLearning
------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/deepLearning
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: deep learning

`BACK TO TOP <AnalysisTechnique_>`_

------------

densityMeasurement
------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/densityMeasurement
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: density measurement

`BACK TO TOP <AnalysisTechnique_>`_

------------

dictionaryLearning
------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/dictionaryLearning
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Dictionary learning' is a branch of signal processing and machine learning that aims at finding a frame (called dictionary) in which some training data admits a sparse representation.
   :name: dictionary learning

`BACK TO TOP <AnalysisTechnique_>`_

------------

diffeomorphicRegistration
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/diffeomorphicRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Diffeomorphic registration' refers to a suite of algorithms that register or build correspondences between dense coordinate systems in medical imaging by ensuring the solutions are diffeomorphic.
   :name: diffeomorphic registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

enzymeLinkedImmunosorbentAssay
------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/enzymeLinkedImmunosorbentAssay
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The 'enzyme-linked immunosorbent assay' is a commonly used analytical biochemistry assay for the quantitative determination of antibodies, first described by [Engvall and Perlmann (1972)](https://www.jimmunol.org/content/109/1/129.abstract). [adapted from [wikipedia](https://en.wikipedia.org/wiki/ELISA)]
   :description: This immunoassay utilizes an antibody labeled with an enzyme marker such as horseradish peroxidase. While either the enzyme or the antibody is bound to an immunosorbent substrate, they both retain their biologic activity; the change in enzyme activity as a result of the enzyme-antibody-antigen reaction is proportional to the concentration of the antigen and can be measured spectrophotometrically or with the naked eye. Many variations of the method have been developed.
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0484188
   :name: enzyme-linked immunosorbent assay
   :preferredOntologyIdentifier: http://id.nlm.nih.gov/mesh/2018/M0007526

`BACK TO TOP <AnalysisTechnique_>`_

------------

eyeMovementTracking
-------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/eyeMovementTracking
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Eye movement tracking' refers to a group of techniques used to measure the eye movement and/or position of a living specimen over a given period of time.
   :interlexIdentifier: http://uri.interlex.org/ilx_0417680
   :name: eye movement tracking
   :preferredOntologyIdentifier: http://id.nlm.nih.gov/mesh/2018/M0493574

`BACK TO TOP <AnalysisTechnique_>`_

------------

geneExpressionMeasurement
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/geneExpressionMeasurement
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: gene expression measurement

`BACK TO TOP <AnalysisTechnique_>`_

------------

generalLinearModeling
---------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/generalLinearModeling
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: general linear modeling

`BACK TO TOP <AnalysisTechnique_>`_

------------

geneticCorrelationAnalysis
--------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/geneticCorrelationAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: genetic correlation analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

geneticRiskScore
----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/geneticRiskScore
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A genetic risk score is an estimate of the cumulative contribution of genetic factors to a specific outcome of interest in an individual (Igo et al, 2019).
   :description: [described in: Igo, R. P., Jr, Kinzy, T. G., & Cooke Bailey, J. N. (2019). Genetic Risk Scores. Current protocols in human genetics, 104(1), e95. https://doi.org/10.1002/cphg.95]
   :name: genetic risk score

`BACK TO TOP <AnalysisTechnique_>`_

------------

genomeWideAssociationStudy
--------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/genomeWideAssociationStudy
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'genome-wide association study' is an analysis technique comparing the allele frequencies of all available (or a whole genome representative set of) polymorphic markers in unrelated individuals with a specific symptom or disease condition, and those of healthy controls to identify markers associated with a specific disease or condition.
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0104603
   :knowledgeSpaceLink: https://knowledge-space.org/wiki/NLXINV:1005075#genome-association-studies
   :name: genome-wide association study
   :preferredOntologyIdentifier: http://edamontology.org/topic_3517

`BACK TO TOP <AnalysisTechnique_>`_

------------

globalSignalRegression
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/globalSignalRegression
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'global signal regression' is a denoising technique where the global signal is removed from the time series of each voxel through linear regression. [adapted from: [Murphy & Fox, 2017](https://dx.doi.org/10.1016%2Fj.neuroimage.2016.11.052)]
   :name: global signal regression

`BACK TO TOP <AnalysisTechnique_>`_

------------

hierarchicalAgglomerativeClustering
-----------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/hierarchicalAgglomerativeClustering
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: hierarchical agglomerative clustering

`BACK TO TOP <AnalysisTechnique_>`_

------------

hierarchicalClustering
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/hierarchicalClustering
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: hierarchical clustering

`BACK TO TOP <AnalysisTechnique_>`_

------------

hierarchicalDivisiveClustering
------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/hierarchicalDivisiveClustering
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: hierarchical divisive clustering

`BACK TO TOP <AnalysisTechnique_>`_

------------

imageDistortionCorrection
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/imageDistortionCorrection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Image distortion correction' is the general term for any image processing technique correcting optical or perspective aberrations of an image.
   :name: image distortion correction

`BACK TO TOP <AnalysisTechnique_>`_

------------

imageRegistration
-----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/imageRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: An 'image registration' is a process of bringing a set of images into the same coordinate system.
   :name: image registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

independentComponentAnalysis
----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/independentComponentAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: independent component analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

interSubjectAnalysis
--------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/interSubjectAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: inter-subject analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

interpolation
-------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/interpolation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: An 'interpolation' is an analysis technique that delivers estimates for new data points based on a range of a discrete set of known data points.
   :name: interpolation

`BACK TO TOP <AnalysisTechnique_>`_

------------

intraSubjectAnalysis
--------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/intraSubjectAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: intra-subject analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

k-meansClustering
-----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/k-meansClustering
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'k-means clustering' is a centroid-based cluster analysis technique that aims to partition n observations into a pre-defined number of k clusters by assigning each observation to the cluster with the nearest mean (centroid).
   :name: k-means clustering

`BACK TO TOP <AnalysisTechnique_>`_

------------

linearImageRegistration
-----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/linearImageRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'linear image registration' is a process of bringing a set of images into the same coordinate system using linear transformation.
   :name: linear image registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

linearRegression
----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/linearRegression
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'linear regression' is an analysis approach for modelling the linear relationship between a scalar response and one or more explanatory variables.
   :name: linear regression

`BACK TO TOP <AnalysisTechnique_>`_

------------

linearTransformation
--------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/linearTransformation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'linear transformation' is a linear mathematical function to map coordinates between two different coordinate systems while preserving straight lines.
   :name: linear transformation

`BACK TO TOP <AnalysisTechnique_>`_

------------

literatureMining
----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/literatureMining
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: literature mining

`BACK TO TOP <AnalysisTechnique_>`_

------------

manifoldLearning
----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/manifoldLearning
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'manifold learning' refers to a group of machine learning algorithms for non-linear dimensionality reduction of high-dimensionalty data.
   :name: manifold learning

`BACK TO TOP <AnalysisTechnique_>`_

------------

massUnivariateAnalysis
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/massUnivariateAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'mass univariate analysis' is the statistical analysis of a massive number of simultaneously measured dependent variables via the performance of univariate hypothesis tests.
   :name: mass univariate analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

maximumLikelihoodEstimation
---------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/maximumLikelihoodEstimation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Maximum likelihood estimation' is a statistical analysis technique that estimates the parameters of an assumed probability distribution for some observed data by maximizing a likelihood function so that, under the assumed statistical model, the observed data is most probable. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Maximum_likelihood_estimation)]
   :name: maximum likelihood estimation technique

`BACK TO TOP <AnalysisTechnique_>`_

------------

maximumProbabilityProjection
----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/maximumProbabilityProjection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: maximum probability projection

`BACK TO TOP <AnalysisTechnique_>`_

------------

metaAnalysis
------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/metaAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: meta-analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

metaAnalyticConnectivityModeling
--------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/metaAnalyticConnectivityModeling
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: meta-analytic connectivity modeling

`BACK TO TOP <AnalysisTechnique_>`_

------------

metadataParsing
---------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/metadataParsing
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: metadata parsing

`BACK TO TOP <AnalysisTechnique_>`_

------------

modelBasedStimulationArtifactCorrection
---------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/modelBasedStimulationArtifactCorrection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The 'model-based stimulation artifact correction' is a model-based analysis technique for removing stimulation artifacts from intracranial electroencephalography signals to uncover the cortico-cortical evoked potentials caused by the stimulation (cf. [Trebaul et al. (2016)](https://doi.org/10.1016/j.jneumeth.2016.03.002)).
   :name: model-based stimulation artifact correction

`BACK TO TOP <AnalysisTechnique_>`_

------------

morphometricAnalysis
--------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/morphometricAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: morphometric analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

morphometry
-----------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/morphometry
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: morphometry

`BACK TO TOP <AnalysisTechnique_>`_

------------

motionCorrection
----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/motionCorrection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Motion correction' is the general term for any preprocessing analysis technique used to correct for motion artifacts in imaging time-series.
   :name: motion correction

`BACK TO TOP <AnalysisTechnique_>`_

------------

movementTracking
----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/movementTracking
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Movement tracking' refers to a group of techniques used to measure the movement and/or position of an object, specimen, or anatomical parts of a specimen over a given period of time.
   :name: movement tracking

`BACK TO TOP <AnalysisTechnique_>`_

------------

multi-compartmentModeling
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/multi-compartmentModeling
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: multi-compartment modeling

`BACK TO TOP <AnalysisTechnique_>`_

------------

multi-scaleIndividualComponentClustering
----------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/multi-scaleIndividualComponentClustering
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'multi-scale individual component clustering' is a multi-scale, unsupervised cluster analysis technique to group individual, independent components of a single-object/single-subject independent component analysis (ICA) from an object-pool/subject-pool (cf. [Naveau et al, 2012](https://doi.org/10.1007/s12021-012-9145-2)).
   :name: multi-scale individual component clustering

`BACK TO TOP <AnalysisTechnique_>`_

------------

multiVoxelPatternAnalysis
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/multiVoxelPatternAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'multi-voxel pattern analysis' is considered as a supervised classification problem where a classifier attempts to capture the relationships between spatial patterns of functional magnetic resonance imaging activity and experimental conditions ([Mahmoudi et al., 2012](https://doi.org/10.1155/2012/961257), [Davatzikos et al., 2005](https://doi.org/10.1016/j.neuroimage.2005.08.009)).
   :name: multi-voxel pattern analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

multipleLinearRegression
------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/multipleLinearRegression
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'multiple linear regression' is a linear approach for modelling the relationship between a scalar response and multiple explanatory variables. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Linear_regression)]
   :name: multiple linear regression

`BACK TO TOP <AnalysisTechnique_>`_

------------

neuromorphicSimulation
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/neuromorphicSimulation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: neuromorphic simulation

`BACK TO TOP <AnalysisTechnique_>`_

------------

nonlinearImageRegistration
--------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/nonlinearImageRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'nonlinear image registration' is a process of bringing a set of images into the same coordinate system using nonlinear transformation.
   :name: nonlinear image registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

nonlinearTransformation
-----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/nonlinearTransformation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'nonlinear transformation' is a mathematical function to map coordinates between two different coordinate systems, not preserving straight lines.
   :name: nonlinear transformation

`BACK TO TOP <AnalysisTechnique_>`_

------------

nonrigidImageRegistration
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/nonrigidImageRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'nonrigid image registration' is a process of bringing a set of images into the same coordinate system using nonrigid transformation.
   :name: nonrigid image registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

nonrigidMotionCorrection
------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/nonrigidMotionCorrection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: nonrigid motion correction

`BACK TO TOP <AnalysisTechnique_>`_

------------

nonrigidTransformation
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/nonrigidTransformation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'nonrigid transformation' is a specific linear transformation using combinations of rotations, translations, reflections, scaling, shearing, and perspective projections to map coordinates between two coordinate spaces.
   :name: nonrigid transformation

`BACK TO TOP <AnalysisTechnique_>`_

------------

nuisanceRegression
------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/nuisanceRegression
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Nuisance regression' is an image processing technique which seeks to attenuate non-neural BOLD fluctuations from measurable noise sources such as scanner drift and head motion, as well as periodic physiological signals. [adapted from [Hallquist et al. 2013](https://doi.org/10.1016%2Fj.neuroimage.2013.05.116)]
   :name: nuisance regression

`BACK TO TOP <AnalysisTechnique_>`_

------------

pathwayAnalysis
---------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/pathwayAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'pathway analysis' refers to a group of techniques that aim to discover what biological themes, and which biomolecules, are crucial to understand biological pathways of (typically) high-throughput biological data (adapted from [García-Campos et al., 2015](https://doi.org/10.3389/fphys.2015.00383)).
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0778897
   :name: pathway analysis
   :preferredOntologyIdentifier: http://edamontology.org/operation_3928

`BACK TO TOP <AnalysisTechnique_>`_

------------

performanceProfiling
--------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/performanceProfiling
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: performance profiling

`BACK TO TOP <AnalysisTechnique_>`_

------------

perturbationalComplexityIndexMeasurement
----------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/perturbationalComplexityIndexMeasurement
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: perturbational complexity index measurement

`BACK TO TOP <AnalysisTechnique_>`_

------------

phaseSynchronizationAnalysis
----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/phaseSynchronizationAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'phase synchronization analysis' detects and quantifies synchronization between two time series.
   :name: phase synchronization analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

principalComponentAnalysis
--------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/principalComponentAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'principal component analysis' is a statistical technique for reducing the dimensionality of a dataset by linearly transforming the data into a new coordinate system where (most of) the variation in the data can be described with fewer dimensions than the initial data. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Principal_component_analysis)]
   :name: principal component analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

probabilisticAnatomicalParcellationTechnique
--------------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/probabilisticAnatomicalParcellationTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: probabilistic anatomical parcellation technique

`BACK TO TOP <AnalysisTechnique_>`_

------------

probabilisticDiffusionTractography
----------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/probabilisticDiffusionTractography
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: probabilistic diffusion tractography

`BACK TO TOP <AnalysisTechnique_>`_

------------

pupillometry
------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/pupillometry
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: Pupillometry is the measurement of minute fluctuations in pupil diameter in response to a stimulus.
   :name: pupillometry

`BACK TO TOP <AnalysisTechnique_>`_

------------

qualitativeAnalysis
-------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/qualitativeAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Qualitative analysis' uses subjective judgment to analyze data based on non-quantifiable information. The resulting data are typically nonnumerical.
   :name: qualitative analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

quantification
--------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/quantification
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: quantification

`BACK TO TOP <AnalysisTechnique_>`_

------------

quantitativeAnalysis
--------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/quantitativeAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: quantitative analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

reconstructionTechnique
-----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/reconstructionTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'reconstruction technique' is able to re-build, re-assemble, re-create, or re-imagine something by applying (often mathematical) principles to physical evidence.
   :name: reconstruction technique

`BACK TO TOP <AnalysisTechnique_>`_

------------

reporterGeneBasedExpressionMeasurement
--------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/reporterGeneBasedExpressionMeasurement
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: reporter gene based expression measurement

`BACK TO TOP <AnalysisTechnique_>`_

------------

reporterProteinBasedExpressionMeasurement
-----------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/reporterProteinBasedExpressionMeasurement
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: reporter protein based expression measurement

`BACK TO TOP <AnalysisTechnique_>`_

------------

rigidImageRegistration
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/rigidImageRegistration
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'rigid image registration' is a process of bringing a set of images into the same coordinate system using rigid transformation.
   :name: rigid image registration

`BACK TO TOP <AnalysisTechnique_>`_

------------

rigidMotionCorrection
---------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/rigidMotionCorrection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: rigid motion correction

`BACK TO TOP <AnalysisTechnique_>`_

------------

rigidTransformation
-------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/rigidTransformation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'rigid transformation' is a specific linear transformation using combinations of rotations, translations, and reflections to map coordinates between two coordinate spaces, leaving the object congruent.
   :name: rigid transformation

`BACK TO TOP <AnalysisTechnique_>`_

------------

rule-basedModeling
------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/rule-basedModeling
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: rule-based modeling

`BACK TO TOP <AnalysisTechnique_>`_

------------

seed-basedCorrelationAnalysis
-----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/seed-basedCorrelationAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: seed-based correlation analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

semanticAnchoring
-----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/semanticAnchoring
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: semantic anchoring

`BACK TO TOP <AnalysisTechnique_>`_

------------

semiquantitativeAnalysis
------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/semiquantitativeAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: An analysis technique which constitutes or involves less than quantitative precision.
   :name: semiquantitative analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

signalFilteringTechnique
------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/signalFilteringTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Signal filtering' is a signal processing technique used to remove or suppress unwanted components or features (e.g., certain frequencies) from a measured signal. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Filter_(signal_processing))]
   :interlexIdentifier: http://uri.interlex.org/ilx_0739623
   :name: signal filtering technique
   :preferredOntologyIdentifier: http://uri.interlex.org/tgbugs/uris/indexes/ontologies/methods/151

`BACK TO TOP <AnalysisTechnique_>`_

------------

signalProcessingTechnique
-------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/signalProcessingTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Signal processing' refers to a class of analysis techniques used to improve transmission, storage efficiency and subjective quality as well as to emphasize or detect components of interest in a measured signal. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Signal_processing)]
   :interlexIdentifier: http://uri.interlex.org/ilx_0739633
   :name: signal processing technique
   :preferredOntologyIdentifier: http://uri.interlex.org/tgbugs/uris/readable/technique/sigproc

`BACK TO TOP <AnalysisTechnique_>`_

------------

simulation
----------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/simulation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: simulation

`BACK TO TOP <AnalysisTechnique_>`_

------------

singleCellRNASequencing
-----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/singleCellRNASequencing
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: single cell RNA sequencing

`BACK TO TOP <AnalysisTechnique_>`_

------------

singleGeneAnalysis
------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/singleGeneAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'single gene analysis' is a genetic test (sequencing technique) to check for any genetic changes in a specific gene.
   :name: single gene analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

singleNucleotidePolymorphismDetection
-------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/singleNucleotidePolymorphismDetection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Single nucleotide polymorphism detection' refers to a group of techniques that are used to scan for new polymorphisms and to determine the allele(s) of a known polymorphism in target sequences (adapted from [Kwok and Chen, 2003](https://doi.org/10.21775/cimb.005.043)).
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0780321
   :name: single nucleotide polymorphism detection
   :preferredOntologyIdentifier: http://edamontology.org/operation_0484

`BACK TO TOP <AnalysisTechnique_>`_

------------

sliceTimingCorrection
---------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/sliceTimingCorrection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Slice timing correction' is a preprocessing technique applied to functional magnetic resonance image data in order to correct for temporal offsets between 2D image slices during the data acquisition. [adapted from [Parker and Razlighi, 2019](https://doi.org/10.3389/fnins.2019.00821)]
   :name: slice timing correction

`BACK TO TOP <AnalysisTechnique_>`_

------------

spectralPowerAutoSegmentationTechnique
--------------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/spectralPowerAutoSegmentationTechnique
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'spectral power auto-segmentation technique' makes use of the power spectrum along the time axis of individual pixels or voxels in an image to automatically generate a segmentation.
   :name: spectral power auto-segmentation technique

`BACK TO TOP <AnalysisTechnique_>`_

------------

spikeSorting
------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/spikeSorting
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Spike sorting' is a class of techniques used in the analysis of extracellular electrophysiological data to extract the activity of one or more neurons from the background electrical noise by making use of the typical waveforms action potentials (spikes) create in the recorded neuronal signal.
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0739628
   :name: spike sorting

`BACK TO TOP <AnalysisTechnique_>`_

------------

stochasticOnlineMatrixFactorization
-----------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/stochasticOnlineMatrixFactorization
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Stochastic online matrix factorization' is a matrix-factorization algorithm that scales to input matrices with both huge number of rows and columns [(Mensch et al., 2018)](https://doi.org/10.1109/TSP.2017.2752697).
   :name: stochastic online matrix factorization

`BACK TO TOP <AnalysisTechnique_>`_

------------

structuralCovarianceAnalysis
----------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/structuralCovarianceAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: structural covariance analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

supportVectorMachineClassifier
------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/supportVectorMachineClassifier
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'support-vector machine classifier' is a supervised machine learning technique that analyzes data for classification.
   :name: support-vector machine classifier

`BACK TO TOP <AnalysisTechnique_>`_

------------

supportVectorMachineRegression
------------------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/supportVectorMachineRegression
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'Support-Vector Regression Algorithm' is a supervised machine learning technique used to estimate the relationship between a dependent and a number of independent variables.
   :name: support-vector regression algorithm

`BACK TO TOP <AnalysisTechnique_>`_

------------

surfaceProjection
-----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/surfaceProjection
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: surface projection

`BACK TO TOP <AnalysisTechnique_>`_

------------

temporalFiltering
-----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/temporalFiltering
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Temporal filtering' is a functional image signal processing technique that aims to remove or attenuate frequencies that vary along the time axis of the raw signal. [adapted from [Wikibooks](https://en.wikibooks.org/wiki/Neuroimaging_Data_Processing/Processing/Steps/Temporal_Filtering)]
   :name: temporal filtering

`BACK TO TOP <AnalysisTechnique_>`_

------------

tractTracing
------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/tractTracing
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: tract tracing

`BACK TO TOP <AnalysisTechnique_>`_

------------

tractography
------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/tractography
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: tractography

`BACK TO TOP <AnalysisTechnique_>`_

------------

transformation
--------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/transformation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: A 'transformation' is a mathematical function to map coordinates between two different coordinate systems.
   :name: transformation

`BACK TO TOP <AnalysisTechnique_>`_

------------

video-oculography
-----------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/video-oculography
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: video-oculography

`BACK TO TOP <AnalysisTechnique_>`_

------------

videoAnnotation
---------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/videoAnnotation
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: video annotation

`BACK TO TOP <AnalysisTechnique_>`_

------------

voxel-basedMorphometry
----------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/voxel-basedMorphometry
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :name: voxel-based morphometry

`BACK TO TOP <AnalysisTechnique_>`_

------------

wholeGenomeSequencing
---------------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/wholeGenomeSequencing
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: 'Whole genome sequencing' is a genetic test (sequencing technique) to determine the entire, or nearly the entire, DNA sequence of an organism's genome at a single time. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Whole_genome_sequencing)]
   :interlexIdentifier: http://uri.interlex.org/base/ilx_0492452
   :name: whole genome sequencing
   :preferredOntologyIdentifier: http://id.nlm.nih.gov/mesh/2018/M000621306

`BACK TO TOP <AnalysisTechnique_>`_

------------

zScoreAnalysis
--------------

.. admonition:: metadata sheet

   :@id: https://openminds.ebrains.eu/instances/technique/zScoreAnalysis
   :@type: https://openminds.ebrains.eu/controlledTerms/Technique
   :definition: The 'z-score analysis' is a statistical normalization technique where the z-score is calculated by subtracting the population mean from an individual raw score (observed data point) and dividing the difference by the population standard deviation. [adapted from [Wikipedia](https://en.wikipedia.org/wiki/Standard_score)]
   :name: z-score analysis

`BACK TO TOP <AnalysisTechnique_>`_

------------

